<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Twister</author>
 <class>frmMain</class>
 <widget class="QDialog" name="frmMain">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>410</width>
    <height>399</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Novelty Assistant</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../novelty_assistant.qrc">
    <normaloff>:/images/main_64.ico</normaloff>:/images/main_64.ico</iconset>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="documentMode">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="tabTasks">
      <attribute name="title">
       <string notr="true">Задачи</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QFrame" name="frame">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>50</height>
          </size>
         </property>
         <property name="frameShape">
          <enum>QFrame::NoFrame</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="cmdNew">
            <property name="toolTip">
             <string>CTRL+N</string>
            </property>
            <property name="text">
             <string>Новая</string>
            </property>
            <property name="shortcut">
             <string>Ctrl+N</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Задачи на дату:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dt">
            <property name="displayFormat">
             <string>dd.MM.yyyy</string>
            </property>
            <property name="calendarPopup">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="cmdRefresh">
            <property name="toolTip">
             <string>F5</string>
            </property>
            <property name="text">
             <string>Обновить</string>
            </property>
            <property name="shortcut">
             <string>F5</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
            <property name="flat">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabTime">
      <attribute name="title">
       <string notr="true">Приход / уход</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout"/>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../novelty_assistant.qrc"/>
 </resources>
 <connections/>
</ui>
